_posts/2020-03-29-newpipe-0.19.0-released.md

```diff
@@ -0,0 +1,71 @@
+
+[@wb9688](https://github.com/wb9688) and [@B0pol](https://github.com/B0pol) (in his second contribution) added share-to-NewPipe support for more URLs. Now `music.youtube.com` and `media.ccc.de` are natively supported. Thanks, you two. In the last point in this section, [@jonas-skywalker](https://github.com/jonas-skywalker) (may the force be with you) improved the search section. Now the magnifying glass vanishes once you start typing.
+
+## Fixes a.k.a. NewPipe crashes less \o/
+
+Android 9 changed how databases behave, effectively destroying backups and exporting remote playlists - not anymore, thanks to [@yausername](https://github.com/yausername). [@XiangRongLin](https://github.com/XiangRongLin) improved sharing a downloaded video to other apps, which crashed the app before this release. The seek duration setting was not properly shown in any other language then English, but [@B0pol](https://github.com/B0pol) let them magically appear again. And finally, [@kapodamy](https://github.com/kapodamy) took care of a nasty bug, which made some WebM videos non seekable. Now you can seek all you want. He also fixed a crash when you tried to download a really large video.
+
+All of the following fixes were done by the incredible [@mauriciocolli](https://github.com/mauriciocolli). Overall, he brought seven (!) different contributions to today's release. He solved an issue where the last screen of the app could be popping up during the app exit animation. A bug in the main screen resulted in tabs not being restored correctly if the app was put to sleep by Android. The main screen had another issue, especially with the menu, which could vanish until the user interacted with the app again. Sharing a YouTube link with NewPipe could end up crashing NewPipe if certain characters were in those links, which is not an issue anymore. And finally, he improved the handling of unavailable content on YouTube, which resulted in a crash previously.
+
+## Development
+
+This release also includes two big improvements of the development process. [@Stypox](https://github.com/Stypox) and [@mauriciocolli](https://github.com/mauriciocolli) changed the way `.apk` files are generated in the development process. This makes it way easier to test a change on your device, since you can have all of these NewPipe apps alongside each other, as long as you have the storage for them.
```

Comment:
```suggestion
This release also includes two big improvements in the development process. [@Stypox](https://github.com/Stypox) and [@mauriciocolli](https://github.com/mauriciocolli) changed the way `.apk` files are generated in the development process. This makes it way easier to test a change on your device, since you can install all of these test versions of Newpipe alongside each other, as long as you have the storage for them.
```